src/Util/ZipFiles.php

```diff
@@ -38,6 +38,7 @@ public function start(array $outputFileStream, string $zipFileName): void
     public function addFile(string $fileName, array $fileStream): void
     {
         if (!empty($fileStream) and is_resource($fileStream['fileStream'])) {
+            $fileName = FileUtilities::fixFileNameLength($fileName);
```

Comment:
I don't think we need this here

Reply:
I have been looking into this some more, and it was said there is no limit, but a limit was made of 250, because of Windows.
PKWare said the limit is 64k Bytes, but that is split by name field, extra field, and comment field.

I guess this can be removed.